src/jupyter_contrib_nbextensions/nbextensions/pyVariableInspector/README.md

```diff
@@ -0,0 +1,25 @@
+# Python Variable Inspector
+
+## Description and main features
+
+The Python Variable Inspector extension enables to collect all defined variables and display them in a floating window. The window not only display the name of variables but also  their type, size in memory and content. The columns are sortable. The window is draggable, resizable, collapsable. The list of displayed variables is automatically updated at each cell execution. Variables can be deleted from workspace by pushing a link. Position and state (displayed/collapsed) are stored in the notebook's metadata and restored at startup. 
+
+#### Demo:
+![](demo.gif)
+
+ 
+## Configuration
+The initial configuration can be given using the IPython-contrib nbextensions facility. It includes:
+
+- Display at startup or not (default: false) 
+- Width of columns (actually the max number of character to display in each column)
+- The state of the toc cell (default: false, ie not present)
```

Comment:
toc cell? I guess this might be a copy-paste hangover? Actually, for all these options, I think it's nice to state the variable name in the config, e.g. 

```markdown
 - `pyVarInspector.types_to_exclude`: The list of types to exclude from display (default:...
```